docs/articles/documentation/reference/api/testcafe/createlivemoderunner.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title: TestCafe.createLiveModeRunner Method
+permalink: /documentation/reference/api/testcafe/createlivemoderunner.html
+---
+# TestCafe.createLiveModeRunner Method
+
+Creates a [test runner](../livemoderunner.md) that runs TestCafe in [live mode](../../../guides/basic-guides/run-tests.md#live-mode). In this mode, TestCafe watches for changes you make in the test files and all files referenced in them (like page objects or helper modules). These changes immediately restart the tests so that you can see the effect.
```

Comment:
> These changes immediately restart the tests so that you can see the effect.

See the effect of what? This sentence should probably be rephrased :)

Reply:
Argh, that's what Dirk was saying all the time. =)

The effect of 'these changes'. I'm not sure that 'their effect' looks nicer though. 'The effect they make' - also bit weird isn't it?

I was trying to rephrase this whole paragraph but no luck. If you can help, that would be groundbreaking for TestCafe Live Mode documentation.

Reply:
"The tests restart immediately after these changes, so you get the real-time updates"

How about something like this?